src/viewer.rs

```diff
@@ -297,6 +294,21 @@ impl GltfViewer {
             println!("Saved {}x{} screenshot to {}", width, height, filename);
         }
     }
+    pub fn multiscreenshot(&mut self, filename: &str, _width: u32, _height: u32, _number: u32) {
+        for x in 1..(_number+1) {
+            let rotate_by_degree : f32 = 6.0 * (x as f32);
```

Comment:
So the angle is always 6 degrees? As far as I understand the meshtool determines the step size based on the number of images: https://github.com/pycollada/meshtool/blob/b52501d310b9be16d46c171c08ff9ff1e0adb61a/meshtool/filters/panda_filters/save_rotate_screenshots.py#L12.

Reply:
The meshtool logic has been implemented now ..